tpm/commands.go

```diff
@@ -143,6 +143,17 @@ func getPubKey(rw io.ReadWriter, keyHandle tpmutil.Handle, ca *commandAuth) (*pu
 	return &pk, &ra, ret, nil
 }
 
+// getCapability reads the requested capability and sub-capability from NVRAM
+func getCapability(rw io.ReadWriter, cap, subcap uint32) ([]byte, error) {
+	var b []byte
+	in := []interface{}{cap, uint32(4), subcap} // tcsd requires capabilities to be of size 4, cannot find any mention of this in the spec
```

Comment:
This seems to contradict the existing use of ordGetCapability 

https://github.com/google/go-tpm/blob/7a7fe86fbbf203a562400a8a0dbe63a93ae02059/tpm/tpm.go#L46

Any thoughts why this is different?

Reply:
It actually doesn't contradict it. The existing use packs the subcapability into a byte slice whereas I have the length and the actual subcap seperate (length+rawBytes). They are both encoded the same way though technically the existing call matches the spec more closely.